nvm.sh

```diff
@@ -2345,11 +2345,11 @@ nvm() {
     return $?
   fi
 
-  argsList=("$@")
-  for i in ${argsList[@]}
+  argsList="$@"
```

Comment:
Should this be  argsList=$@  ?

Reply:
According to https://github.com/koalaman/shellcheck/wiki/SC2124, the recommended code is:
`var=( "$@" )
for i in "${var[@]}"; do ..; done`

Perhaps the spaces are necessary within the parentheses. There were no spaces the first time: `argsList=("$@")`

Reply:
Thanks @gitburd! Simplifying it to `for i in $@` got it to pass!